hermes/webapp/src/index.html

```diff
@@ -12,10 +12,11 @@
         <div id="header">Hermes</div>
         <div style="padding: 10px" ng-view></div>
     </div>
-
     <script src="/vendor/angular/angular.js"></script>
     <script src="/vendor/angular-route/angular-route.js"></script>
     <script src="/vendor/angular-animate/angular-animate.js"></script>
+    <!--<script src="/vendor/sigma.js/sigma.min.js"></script>-->
```

Comment:
Could be missing it but is this intentional? I don't see sigma included anywhere else.